api-examples/filename-check/BUILD

```diff
@@ -0,0 +1,29 @@
+load("//:trlc.bzl", "trlc_requirements", "trlc_requirements_test", "trlc_specification")
+
+trlc_specification(
+    name = "spec",
+    srcs = ["example.rsl"],
+    visibility = ["//api-examples:__subpackages__"],
+)
+
+trlc_requirements(
+    name = "reqs",
+    srcs = ["example.trlc"],
+    spec = [":spec"],
```

Comment:
Why not `spec = ["example.rsl"],`?

Reply:
We want to be able to have the `spec` in a central place an utilize it at different places. If you would allow a direct file dependency here, you would need to expose the `.rsl` file via export_files, which is a code smell.

Having it separated into different targets makes it more modular and reusable.